runtime/rasdump/TextFileStream.cpp

```diff
@@ -51,7 +51,11 @@ void
 TextFileStream::open(const char* fileName, bool cacheWrites)
 {
 	PORT_ACCESS_FROM_PORT(_PortLibrary);
-	if (fileName[0] != '-' ) {
+	if (NULL != strstr(fileName, "/STDOUT/")) {
+		_FileHandle = 1; 
+	} else if (NULL != strstr(fileName, "/STDERR/")) {
+		_FileHandle = 2;
```

Comment:
* we should be using the constants `OMRPORT_TTY_OUT` and `OMRPORT_TTY_ERR` which also avoids problems on platforms where `FD_BIAS != 0`

* we should not close `_FileHandle` for stdout or stderr (because we did not open it)

Reply:
*  Where exactly are`OMRPORT_TTY_OUT` and `OMRPORT_TTY_ERR` defined? As well @pshipton, is this change necessary if all of the grinder instances passed?

Reply:
Those macros are defined in `omr/include_core/omrport.h`.

A fix to not close stdout or stderr is necessary.

Reply:
Yes, the change to use OMRPORT_TTY_OUT and OMRPORT_TTY_ERR should also be done, in case something changes in the future.

Reply:
It would also be a good idea to create #define's for `/STDOUT/` and `/STDERR/` and use the defines instead of the strings.